dcos/package.py

```diff
@@ -24,28 +24,29 @@ def uninstall(pkg, package_name, remove_all, app_id, cli, app):
     :rtype: None
```

Comment:
The docstring is wrong and doesn't include `cli` and `app` arguments, but does include a (nonexistent) `init_client` argument. Can you please update this to include the correct information.